alcs/acs2/PMark.py

```diff
@@ -2,13 +2,15 @@
 
 from alcs.acs2 import Condition
 
-from alcs.acs2 import Constants as c
+from alcs.acs2 import default_configuration
 from alcs import Perception
+from alcs.acs2.ACS2Configuration import ACS2Configuration
 
 
 class PMark(list):
-    def __init__(self):
-        list.__init__(self, [set() for _ in range(c.CLASSIFIER_LENGTH)])
+    def __init__(self, cfg: ACS2Configuration = default_configuration):
+        self.cfg = default_configuration
```

Comment:
You are always overwriting the configuration here

Reply:
Yes, will fix.